plugins/rtmp-services/data/schema/service-schema-v3.json

```diff
@@ -110,7 +110,8 @@
                                 "enum": [
                                     "rtmp_output",
                                     "ffmpeg_hls_muxer",
-                                    "ftl_output"
+                                    "ftl_output",
+                                    "ffmpeg_mpegts_muxer"
```

Comment:
Depends on how older version would handle this. In general it shouldn't be too big of an issue since the number if users on old versions or using a service other than the main big ones isn't too large. But if it turns out to be too big of a pain point we can introduce a new version (v4) of the services file.

Reply:
> Depends on how older version would handle this. 

None, handle this.

> But if it turns out to be too big of a pain point we can introduce a new version (v4) of the services file.

I have always https://github.com/obsproject/rfcs/pull/39 in mind, so I don't know if bumping is worth it, if the overhaul will happen.

Reply:
ok i'll turn it into v4

Reply:
Bumping doesn't really cost us anything, and this is exactly what it exists for.